docs/pages/desktop-access/getting-started.mdx

```diff
@@ -27,104 +25,123 @@ This guide requires you to have:
 - A Linux server to run the Teleport Desktop Access service on
   - You can reuse an existing server running any other Teleport instance
 
-## Step 1/6: Create a User, Group, and Group Policy Object
+## Step 1/6. Create a restrictive service account `User`.
 
 Teleport requires a service account to connect to your Active Directory domain. We recommend creating a dedicated service account with restrictive permissions
-(described below) for maximal security.
+for maximal security.
 
-### Create a User and Group
+To create the service account, open a PowerShell prompt and copy-paste in the commands below. Be sure to note down the password you choose (you will be prompted to
+enter it in the PowerShell), as you will need to provide it to the Teleport configuration yaml later.
 
-From the start menu of your Domain Controller, search for “Active Directory Users and Computers” and open the corresponding program.
+```powershell
+$Name="Teleport Service Account"
+$SamAccountName="svc-teleport"
+New-ADUser `
```

Comment:
Are these '`' characters intentional? Is that how you wrap lines in powershell?

Reply:
[Indeed](https://stackoverflow.com/a/2608186/6277051)

Reply:
Wild 🤯